tekton/predicates.go

```diff
@@ -39,8 +40,7 @@ func BuildPipelineRunFinishedPredicate() predicate.Predicate {
 			return false
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			return (IsBuildPipelineRun(e.ObjectNew)) &&
-				hasPipelineRunBeenChangedToSigned(e.ObjectOld, e.ObjectNew) // only finished pipelines are signed
+			return IsBuildPipelineRun(e.ObjectNew) && isPipelineRunSigned(e.ObjectNew) && helpers.HasPipelineRunSucceeded(e.ObjectNew)
```

Comment:
Also, can we change the name of this predicate to something like `BuildPipelineRunSignedAndSucceededPredicate()`

Reply:
So now that we're only processing the signed & succeeded build PLRs, should the test logic be updated [here](https://github.com/redhat-appstudio/e2e-tests/blob/f25ebe3fbac695690975539054361aeeb11e5a6f/tests/integration-service/integration.go#L185) to `assertBuildPipelineRunSignedAndSucceeded()`?

Reply:
Yes, I think that would be a good idea.

Reply:
Okay, I'll create a separate task for this.